developer/installation/docker-development.md

```diff
@@ -117,3 +117,7 @@ docker image prune
 ```
 
 If you have big problems, there is a "Reset" option in the Docker for Mac or Docker for Windows app preferences UI.
+
+### Troubleshooting
+
+If you are experiencing issues, first make sure that you are allowing Docker sufficient memory to run. In your Docker preferences, we suggest adjusting the `Memory` setting to allow `3.0GiB`, and the `Swap` setting to allow `1.5GiB`.
```

Comment:
@kieckhafer would you add the screenshot showing your example settings?

Reply:
added